Improve the template

ItsPierre opened the pull request:
Add more and better examples.
Change some settings for better usability.

jojomatik commented on components/ui/PasswordInputField.vue:
```diff
@@ -0,0 +1,35 @@
+   */
+  modelValue: string | null;
+  /**
+   * An optional, deviating label for this field.
```
deviating? optional?

ItsPierre replied:
ah, thats deprecated, I will remove it

jojomatik commented on pages/admin/index.vue:
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="d-flex align-center justify-center" style="height: 100vh">
```
why vh?

ItsPierre replied:
because it should take the full height of the screen

jojomatik replied:
You used `dvh` in another place, why the difference?

ItsPierre replied:
ah, because I was not sure, if we should implement such new CSS properties, that some IDEs and some browsers don't support.

But I updated it to `dvh` since the whole template uses the latest features.

jojomatik commented:
According to this old issue https://github.com/nuxt-community/vuetify-module/issues/165 `v-spacer` should only be used in the grid system (https://vuetifyjs.com/en/components/grids/#grid-system).  Should we use `<div class="flex-grow-1"></div` instead?

jojomatik commented:
Also, could you please add seo meta for these pages?

ItsPierre commented:
I removed the `v-spacer` and added a simple div with `flex-grow-1`.

I also added SEO Meta for the home and login page.
Since I am not a professional at SEO I tried my best to use it as it makes sense to me. I don't think you would add SEO meta to the admin page since you don't want this page to be indexed by the search machine.

jojomatik commented on locales/de.json:
```diff
@@ -1,20 +1,24 @@
+  "seo_home_title": "Nuxt 3 Base Repository von jojomatik",
+  "seo_home_description": "Der einfachste Weg, um dein Projekt zu starten.",
+  "seo_auth_login_title": "Anmelden",
+  "seo_auth_login_description": "Melde Dich gleich mit deinem Pseudo-Login ein."
```
```suggestion
  "seo_auth_login_description": "Melde Dich gleich mit deinem Pseudo-Login an."
```

ItsPierre replied:
fixed

jojomatik commented on pages/auth/login.vue:
```diff
@@ -0,0 +1,69 @@
+  ogTitle: i18n.t("seo_auth_login_title"),
+  description: i18n.t("seo_auth_login_description"),
+  ogDescription: i18n.t("seo_auth_login_description"),
+  ogImage: "/assets/jojomatik.png",
```
Is it necessary to define this in every page? Or would it take the one from `app.vue` instead? (the one you removed)

ItsPierre replied:
What do you mean?

I moved it from `app.vue` to the `index.vue`, because the title and description of the `app.vue` / home page should not be applied to the other pages like login etc.
